src/devices/MotorHat/samples/DCMotorSample.cs

```diff
@@ -0,0 +1,52 @@
+﻿using Iot.Device.DCMotor;
+using System;
+using System.Diagnostics;
+using System.Threading;
+
+namespace MotorHat.Samples
+{
+    internal class DCMotorSample
+    {
+        internal static void Run()
+        {
+            const double Period = 10.0;
+            Stopwatch sw = Stopwatch.StartNew();
+
+
+            // Use the following code to generate an I2C address different from the default
+            //var busId = 1;
+            //var selectedI2cAddress = 0b000000;     // A5 A4 A3 A2 A1 A0
+            //var deviceAddress = MotorHat.I2cAddressBase + selectedI2cAddress;
```

Comment:
might be worth adding one overload which takes selectedI2cAddress considering this is hard wired on the hat (make sure it's not ambiguous with frequency overload)